migration-toolkit-for-containers/migrating-applications-with-mtc.adoc

```diff
@@ -60,11 +60,12 @@ include::modules/migration-migrating-applications-api.adoc[leveloffset=+2]
 include::modules/migration-mtc-cr-manifests.adoc[leveloffset=+2]
 
 [id="migrating-applications-mtc-cli-additional-resources_{context}"]
-=== Additional resources
+== Additional resources
```

Comment:
```suggestion
=== Additional resources
```
Based on the id `migrating-applications-mtc-cli-additional-resources_` it seems like this should be a child of *Migrating your applications from the command line*, but it's a sibling of it instead. I might be wrong but wanted to flag this just in case.